### PR TITLE
Episode 8: Zola

### PR DESCRIPTION
**Pull request by bstrie**

### CODE REVIEW

File: _episodes/007-zola.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+length: "52224834"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+[Vincent Prouillet](https://www.vincentprouillet.com/) talks about his experience building the Zola static site generator and reflects back on five years of work with Rust.
+
+Vincent is the creator of [Zola](https://www.getzola.org) (formerly known as Gutenburg), a static site generator built with [Rust](https://www.rust-lang.org/) and [Tera](https://github.com/Keats/tera), a Jinja2-like template engine.
```
Comment (jonhoo) [inline _episodes/007-zola.md]: I think it was called Gutenb**e**rg?

File: _episodes/007-zola.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+If you create a site with Zola, Vincent would appreciate you adding your site to the [EXAMPLES](https://github.com/getzola/zola/blob/master/EXAMPLES.md) file in the repository.
```
Comment (jonhoo) [inline _episodes/007-zola.md]: I don't feel like this belongs in the release notes?

File: _episodes/007-zola.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+#### Other links
+- [Forestry](https://forestry.io/) (WYSIWYG CMS for Static Sites)
+- [Keyword Arguments RFC](https://github.com/rust-lang/rfcs/issues/323)
+- [kickstart](https://github.com/Keats/kickstart) (Scaffolding tool)
+
+### Credits
+
+Intro Theme: [Aerocity](https://twitter.com/AerocityMusic)
+
+Audio Editing: [Jeremy Jung]()
```
Comment (jonhoo) [inline _episodes/007-zola.md]: Missing link

File: _episodes/007-zola.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+Hosting Infrastructure: [Jon Gjengset](https://twitter.com/jonhoo/)
+
+Show Notes: [Ben Striegel](https://twitter.com/bstrie)
+
+Hosts: Jeremy Jung
```
Comment (jonhoo) [inline _episodes/007-zola.md]: Link?

**Comment by bstrie** [comment]
Comments addressed.

File: _episodes/007-zola.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e007-zola-vincent-prouillet.mp3
+duration: "54:24"
+length: "52224834"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+[Vincent Prouillet](https://www.vincentprouillet.com/) talks about his experience building the Zola static site generator and reflects back on five years of work with Rust.
```
Comment (jonhoo) [inline _episodes/007-zola.md]: work**ing**

**Pull request closed without merging**